admin/kernel-masterdata-service/docs/Configuration.md

```diff
@@ -0,0 +1,60 @@
+# Masterdata services Configuration Guide
+## Overview
+The guide here lists down some of the important properties that may be customised for a given installation. Note that the listing here is not exhaustive, but a checklist to review properties that are likely to be different from default.  If you would like to see all the properites, then refer to the files listed below.
+
+## Configuration files
+Masterdata services uses the following configuration files:
+```
+application-default.properties
+kernel-default.properties
+```
+The above files are located in [mosip-config](https://github.com/mosip/mosip-config/blob/develop2-v2/) repo
+
+## DB
+* `masterdata_database_username`
+* `masterdata_database_password`
+* `masterdata_database_url`
+
+Point the above to your DB .  Default is set to point to in-cluster Postgres installed with sandbox.
+
+## Admin UI config
+* `mosip.admin.ui.configs`
+Above property is used for UI configuration of admin. Value of the property is a key value pairs. Example:
+
+* `mosip.admin.ui.configsversion=version:${application.configuration.level.version};HierarchyLevel:${mosip.recommended.centers.locCode}`
```

Comment:
it is important to mention about language-related properties which is important for admin-UI

Reply:
This is just an example so which ever properties are required for UI Configuration those needs to be added as a part of configs